### DISCUSSION TITLE
1ο Θέμα

### DISCUSSION THREAD

**Discussion by ghost** [sdy60/syllabus#86] [Q&A] [answered]
Η εφαρμογή διαχειρίζεται μία φάρμα 5 θέσεων ή ένα δίκτυο από φάρμες 5 θέσεων;

**Comment by epidrome** [comment, marked as answer]
προφανώς ένα δίκτυο από φάρμες 5 θέσεων ώστε να υπάρχουν εναλλακτικές αν κάποιες είναι γεμάτες
